waspc/data/Generator/templates/react-app/src/auth/forms/Auth.tsx

```diff
@@ -497,7 +497,7 @@ const VerifyEmailForm = ({ isLoading, setIsLoading, setErrorMessage, setSuccessM
   const location = useLocation()
   const token = new URLSearchParams(location.search).get('token')
 
-  const submitForm = useCallback(async () => {
+  async function submitForm() {
```

Comment:
`useCallback` was missing a second argument so I decided to remove it. @infomiho FYI